CMakeLists.txt

```diff
@@ -345,9 +346,6 @@ if(enable-examples)
                 examples/plugins/cxx/override/overrideplugin.cpp
                 examples/plugins/cxx/override/overrideplugin.h)
     target_link_libraries(cxxoverrideplugin maliit-plugins Qt5::Widgets)
```

Comment:
I think you need to also remove the `cxxoverrideplugin` from examples here, since you've removed the files in the one commit?

Reply:
I removed unused qml override files, didn't touch the c++ one.